data/mods/CrazyCataclysm/crazy_items.json

```diff
@@ -304,7 +304,7 @@
     "flags": [ "FIRE", "LIGHT_310", "CHARGEDIM", "FLAMING", "DURABLE_MELEE", "TRADER_AVOID", "WATER_EXTINGUISH" ]
   },
   {
-    "id": "flu_shot",
+    "id": "flu_shot_fun",
```

Comment:
changing an id will break saves

Reply:
Hmm, If I do the migration, will all of the `flu_shot` be migrated to `flu_shot_fun`?

Reply:
In this case it won't because the id still exists, all of the existing "fun" shots will just "swap gender" (frog joke) to the old version.